pkgs/applications/networking/icloudpd/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
```

Comment:
If you move this file to `pkgs/by-name/ic/icloudpd/package.nix`, you won't need to add it to `all-packages.nix`